addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/internal/AllPlayBindingProperties.java

```diff
@@ -17,25 +17,34 @@
  */
 public class AllPlayBindingProperties {
 
-    private final Integer rewindSkipTimeInSec;
-    private final Integer fastForwardSkipTimeInSec;
+    private final double rewindSkipTimeInSec;
```

Comment:
Hi, thanks for the feedback!
I just thought double would leave more flexibility to the user. I can easily change it to Integer. Out of curiosity: What is the problem with double here?